demo/components/select/SelectDemo.js

```diff
@@ -28,9 +28,11 @@ export default class SelectDemo extends React.Component {
 
     handleChange(item) {
         if (item === 3) {
-            this.setState({
-                open: false
-            });
+            setTimeout(() => {
+                this.setState({
+                    open: false
+                });
+            },1000);
```

Comment:
It is demo, it doesn't really matter what we do there (at least for now...)
I assume it was used just to check that change works.